lib/commands/isVisibleWithinViewport.js

```diff
@@ -55,17 +55,15 @@ module.exports = function isVisibleWithinViewport (selector) {
     }
 
     return resultPromise.then((res) => {
-        if (Array.isArray(res) && res.length === 1) {
-            return res[0]
-        }
-
-        return res
+        if (Array.isArray(res)) {
+            if (res.length === 1) { return res[0] } else { return res }
+        } else { return false }
```

Comment:
no else needed here

Reply:
But the function will return `undefined` when res is not array?

Reply:
you still can return `false` just not in the else